docs/user/user.md

```diff
@@ -0,0 +1,8 @@
+---
+sidebar_position: 0
+---
+# User
```

Comment:
what is this section?

who is the user here? app dev, end users? a short paragraph explaining what this section is would be needed

Reply:
I think we will have to do this for all sections then. Actually good catch bc it should be User Guides

<img width="1327" alt="Screenshot 2023-12-19 at 17 32 41" src="https://github.com/cosmos/cosmos-sdk-docs/assets/37125168/8552dde1-d650-4785-907f-1766d62d1fde">

Reply:
probably makes sense. In learn and build its a bit better, but this is a bit ambiguous